browser/src/layer/tile/CalcTileLayer.js

```diff
@@ -839,6 +840,14 @@ L.CalcTileLayer = L.CanvasTileLayer.extend({
 			*/
 			this.insertMode = e.state.trim() === '' ? false: true;
 		}
+		else if (e.commandName === '.uno:ToggleSheetGrid') {
+			let newState = e.state.trim() === 'true';
+			if (this._sheetGrid != newState) {
+				this._sheetGrid = newState;
+				// Trigger redraw.
+				this._map.fire('sheetgeometrychanged');
```

Comment:
I am unsure if that's the best event to fire to trigger a complete redraw. I'm afraid it might stop working. I didn't find one to for redraw otherwise.

Reply:
I call `this._painter._sectionContainer.requestReDraw()` directly.